README.md

```diff
@@ -272,9 +272,9 @@ We have also extensively documented working with the [FxA code-base using VS Cod
 
 #### Debugging a server
 
-`npm start` runs some of the services with the debugger enabled by default.
+`yarn start` runs some of the services with the debugger enabled by default.
 
-1. Start the whole server as usual (`npm install && npm start` from top-level in the monorepo)
+1. Start the whole server as usual (`npm install && yarn start` from top-level in the monorepo)
```

Comment:
May as well change `npm install` to `yarn install` (or the shorthand `yarn`)

Reply:
This is what I have used, and I think it works because yarn consumes the same package.json format as npm and can install any package from the npm registry